cmd/virt-launcher/qemu.conf

```diff
@@ -8,3 +8,5 @@ dynamic_ownership = 1
 remember_owner = 0
 namespaces = [ ]
 cgroup_controllers = [ ]
+swtpm_user = "qemu"
```

Comment:
Can we move this to code? The reason being we might not want to run as qemu in the future.

Reply:
I don't think we can. I'm also not sure I understand your concern. If we decide to change the user in the future, can't we just change it here?